.github/workflows/test.yml

```diff
@@ -42,3 +42,16 @@ jobs:
         uses: romeovs/lcov-reporter-action@v0.2.16
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  lighthouse:
+    runs-on: ubuntu-18.04
+    needs: build
+    name: Lighthouse
+    steps:
+      - uses: actions/checkout@v2
+      - uses: browser-actions/setup-chrome@latest
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - run: chrome --version
+      - run: yarn
+      - run: yarn serve && yarn lighthouse
```

Comment:
This will not work. Since serve does not terminate, this will run forever. Consider either using pm2 to manage multiple concurrent processes, or finding a way to run lighthouse on a build folder without deploying it

Reply:
found the config for build folder -- changed in new commit